src/js/utils.js

```diff
@@ -3,6 +3,9 @@ export const setCookie = (value) => {
   d.setTime(d.getTime() + 24 * 60 * 60 * 1000);
   const expires = 'expires=' + d.toUTCString();
   document.cookie = '_cookies_accepted=' + value + '; ' + expires;
+  if (value == 'all' || value == 'functionality') {
```

Comment:
```suggestion
  if ((value == 'all' || value == 'functionality') && typeof dataLayer == 'function') {
```

Reply:
@anthonydillon like this?

Reply:
You just need to check `window.dataLayer` exists before calling it. Otherwise, none GTM usage would error.

So:
```js
if ((value == 'all' || value == 'performance') && typeof dataLayer == 'function') {
```